### PR TITLE
Make Terraform more configurabler

### PR DESCRIPTION
**Pull request by sethvargo**
Make Terraform more configurabler

**Release Note**

```release-note
Make Terraform more configurabler
```

### CODE REVIEW

File: terraform/database.tf [hunk 1]
```diff
@@ -12,18 +12,11 @@
...
-  number  = false
-  upper   = false
-}
-
 resource "google_sql_database_instance" "db-inst" {
   project          = data.google_project.project.project_id
   region           = var.db_location
   database_version = "POSTGRES_11"
-  name             = "en-${random_string.db-name.result}"
+  name             = var.db_name
```
Comment (sethvargo) [inline terraform/database.tf:19]: Note that "name" is ignored on changes, so this won't cause a re-create

File: terraform/vars.tf [hunk 2]
```diff
@@ -42,6 +48,18 @@ variable "kms_location" {
   default = "us-central1"
 }
 
+# Name of the key ring for export signing keys.
+variable "kms_export_signing_key_ring_name" {
```
Comment (chaodaiG) [inline terraform/vars.tf:52]: Do we also want to update their usage in this PR? https://github.com/google/exposure-notifications-server/blob/c1975364369fe6d8b02485656bd3b08c9549d62f/terraform/key_management.tf#L17

File: terraform/database.tf (hunk 1 above)
Comment (icco) [reply terraform/database.tf:19]: Is that documented anywhere? Didn't see it in https://www.terraform.io/docs/providers/google/r/sql_database_instance.html#argument-reference

File: terraform/vars.tf (hunk 2 above)
Comment (sethvargo) [reply terraform/vars.tf:52]: Done

**Comment by chaodaiG** [comment]
lgtm from my end, so
/lgtm
/hold
In case you want to address @icco 's comment

**Comment by icco** [comment]
Given it was currently a random string, doesn't matter beyond my curiosity.

/lgtm

**Pull request merged**